Ghidra/Framework/Generic/src/main/java/resources/ResourceManager.java

```diff
@@ -481,6 +513,8 @@ public static ImageIcon loadImage(String filename, int width, int height) {
 	 * @param filename name of file to load, e.g., "images/home.gif"
 	 * @return the image icon stored in the bytes
 	 */
+
+	 /*
```

Comment:
What's the problem with this comment-out thing? If it is not inteded to be used anymore, it should be deleted.